From 0269f4ebbda27eaa6e1bada6980c60fa1dcb2a19 Mon Sep 17 00:00:00 2001
From: Patrick Combes <patrick.combes@intel.com>
Date: Mon, 5 Aug 2013 16:44:11 +0200
Subject: Configure FwPatchFilePath statically only when needed

BZ: 117478

FwPatchFilePath is configured from bt_vendor.conf when possible and
actually needed.
It is configured at runtime through android properties when the platform
variant is not sufficient.

Change-Id: If0ee232e3970c863174fec3239a49aac5c9d2eac
Signed-off-by: Patrick Combes <patrick.combes@intel.com>
Reviewed-on: http://android.intel.com:8080/126531
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 conf/intel/baylake/bt_vendor.conf      |    2 +-
 conf/intel/bodegabay/bt_vendor.conf    |    3 ---
 conf/intel/byt_m_crb/bt_vendor.conf    |    2 +-
 conf/intel/byt_t_ffrd10/bt_vendor.conf |    2 +-
 conf/intel/byt_t_ffrd8/bt_vendor.conf  |    2 +-
 conf/intel/ctp7160/bt_vendor.conf      |    3 ++-
 conf/intel/ctpscalelt/bt_vendor.conf   |    2 +-
 conf/intel/saltbay_pr0/bt_vendor.conf  |    3 ---
 conf/intel/saltbay_pr1/bt_vendor.conf  |    3 ---
 conf/intel/victoriabay/bt_vendor.conf  |    3 ---
 10 files changed, 7 insertions(+), 18 deletions(-)

diff --git a/conf/intel/baylake/bt_vendor.conf b/conf/intel/baylake/bt_vendor.conf
index 93aa23c..e23e986 100644
--- a/conf/intel/baylake/bt_vendor.conf
+++ b/conf/intel/baylake/bt_vendor.conf
@@ -2,4 +2,4 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
+FwPatchFilePath = /etc/firmware/bt/vv/
diff --git a/conf/intel/bodegabay/bt_vendor.conf b/conf/intel/bodegabay/bt_vendor.conf
index 93aa23c..e6d8154 100644
--- a/conf/intel/bodegabay/bt_vendor.conf
+++ b/conf/intel/bodegabay/bt_vendor.conf
@@ -1,5 +1,2 @@
 # UART device port where Bluetooth controller is attached
 UartPort = /dev/ttyMFD0
-
-# Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
diff --git a/conf/intel/byt_m_crb/bt_vendor.conf b/conf/intel/byt_m_crb/bt_vendor.conf
index 93aa23c..e23e986 100644
--- a/conf/intel/byt_m_crb/bt_vendor.conf
+++ b/conf/intel/byt_m_crb/bt_vendor.conf
@@ -2,4 +2,4 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
+FwPatchFilePath = /etc/firmware/bt/vv/
diff --git a/conf/intel/byt_t_ffrd10/bt_vendor.conf b/conf/intel/byt_t_ffrd10/bt_vendor.conf
index 93aa23c..e23e986 100644
--- a/conf/intel/byt_t_ffrd10/bt_vendor.conf
+++ b/conf/intel/byt_t_ffrd10/bt_vendor.conf
@@ -2,4 +2,4 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
+FwPatchFilePath = /etc/firmware/bt/vv/
diff --git a/conf/intel/byt_t_ffrd8/bt_vendor.conf b/conf/intel/byt_t_ffrd8/bt_vendor.conf
index 93aa23c..e23e986 100644
--- a/conf/intel/byt_t_ffrd8/bt_vendor.conf
+++ b/conf/intel/byt_t_ffrd8/bt_vendor.conf
@@ -2,4 +2,4 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
+FwPatchFilePath = /etc/firmware/bt/vv/
diff --git a/conf/intel/ctp7160/bt_vendor.conf b/conf/intel/ctp7160/bt_vendor.conf
index b06e977..01d8dd2 100644
--- a/conf/intel/ctp7160/bt_vendor.conf
+++ b/conf/intel/ctp7160/bt_vendor.conf
@@ -2,7 +2,8 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
+FwPatchFilePath = /etc/firmware/bt/vv/
+
 LPM_IDLE_TIMEOUT_MULTIPLE = 5
 SCO_USE_I2S_INTERFACE = FALSE
 SCO_I2SPCM_IF_ROLE = 1
diff --git a/conf/intel/ctpscalelt/bt_vendor.conf b/conf/intel/ctpscalelt/bt_vendor.conf
index 93aa23c..e23e986 100644
--- a/conf/intel/ctpscalelt/bt_vendor.conf
+++ b/conf/intel/ctpscalelt/bt_vendor.conf
@@ -2,4 +2,4 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
+FwPatchFilePath = /etc/firmware/bt/vv/
diff --git a/conf/intel/saltbay_pr0/bt_vendor.conf b/conf/intel/saltbay_pr0/bt_vendor.conf
index 93aa23c..e6d8154 100644
--- a/conf/intel/saltbay_pr0/bt_vendor.conf
+++ b/conf/intel/saltbay_pr0/bt_vendor.conf
@@ -1,5 +1,2 @@
 # UART device port where Bluetooth controller is attached
 UartPort = /dev/ttyMFD0
-
-# Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
diff --git a/conf/intel/saltbay_pr1/bt_vendor.conf b/conf/intel/saltbay_pr1/bt_vendor.conf
index 93aa23c..e6d8154 100644
--- a/conf/intel/saltbay_pr1/bt_vendor.conf
+++ b/conf/intel/saltbay_pr1/bt_vendor.conf
@@ -1,5 +1,2 @@
 # UART device port where Bluetooth controller is attached
 UartPort = /dev/ttyMFD0
-
-# Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
diff --git a/conf/intel/victoriabay/bt_vendor.conf b/conf/intel/victoriabay/bt_vendor.conf
index 93aa23c..e6d8154 100644
--- a/conf/intel/victoriabay/bt_vendor.conf
+++ b/conf/intel/victoriabay/bt_vendor.conf
@@ -1,5 +1,2 @@
 # UART device port where Bluetooth controller is attached
 UartPort = /dev/ttyMFD0
-
-# Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
-- 
1.7.4.1

